src/core/opamConsole.ml

```diff
@@ -511,13 +521,13 @@ let clear_status =
 
 let print_message =
   if Sys.win32 then
-    fun ch fmt ->
+    fun ?(stdout=stdout) ch fmt ->
       flush (if ch = `stdout then stderr else stdout);
       clear_status ();
       (* win32_print_message *always* flushes *)
       Printf.ksprintf (win32_print_message ch) fmt
```

Comment:
`win32_print_message` needs a `~stdout` parameter, too (sorry - these functions have ended up a bit of a mess over time... I think just adding that parameter here will be easier than trying to straighten out all the old VT100 emulation code)